marshal/claim_test.go

```diff
@@ -383,6 +383,12 @@ func (s *ClaimSuite) TestHealthCheck(c *C) {
 	c.Assert(s.m.GetPartitionClaim("test16", 0).LastHeartbeat, Equals, int64(0))
 	c.Assert(s.m.GetPartitionClaim("test16", 0).LastOffset, Equals, int64(0))
 	c.Assert(s.m.GetLastPartitionClaim("test16", 0).LastOffset, Equals, int64(21))
+
+	// If we are okay with CV<PV we shouldn't release
+	opts := NewConsumerOptions()
+	opts.ReleaseClaimsIfBehind = false
+	s.cl = newClaim("test16", 0, s.m, s.ch, opts)
```

Comment:
Would this release otherwise? A new claim has the timestamp set to now and will not release even if it hasn't consumed anything, right?